packages/core/src/payment/strategies/stripe-upe/stripe-upe.ts

```diff
@@ -194,7 +194,12 @@ interface ShippingDefaultValues {
 }
 
 interface CustomerDefaultValues {
+    mode?: string;
     email: string;
+    allowedCountries?: any;
```

Comment:
Can we type this better?

Reply:
change the type to string[ instead of any